### ISSUE TITLE
Not an Issue either i just dont know where to ask you so...

### ISSUE THREAD

**Issue by dbstreif** [LWSS/Ape-ex-Abominations#14]
I am a bit confused on how to compile this project because I am used to using Linux. Basically, can you tell me how the hell to compile this cheat and inject it? The readme.md didn't help but I didn't expect it to anyway. By the way I love Fuzion and the great work you have done with it! I hope you can help me.

**Comment by LWSS** [comment]
something like
```
mkdir build
cd build
cmake ..
make
sudo ./Ape-ex*
```
with qemu open and evdev-mirror loaded into kernel

**Comment by dbstreif** [comment]
ok thanks

**Comment by dbstreif** [comment]
what is evdev-mirror because I am familiar with qemu but not evdev. btw I am a noob on windows but unfortunately ubunut cannot run a lot of these games because of anicheat

**Comment by dbstreif** [comment]
yeah its not working. I tried using visual studio but I just cannot build it no matter what I do. I just don't know my way around windows. Thank you for your patience though.

**Comment by LWSS** [comment]
does not require building anything on windows

**Comment by LWSS** [comment]
https://github.com/LWSS/evdev-mirror

**Comment by dbstreif** [comment]
CMake error at CMakeLists.txt:70 (add_executable): No sources given to target: Ape-ex
CMake error at CMakeLists.txt:66 (add_library): No sources given to target: Ape-ex
